tokio-stream/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "tokio-stream"
+# When releasing to crates.io:
+# - Remove path dependencies
+# - Update html_root_url.
+# - Update doc url
+#   - Cargo.toml
+# - Update CHANGELOG.md.
+# - Create "tokio-stream-0.1.x" git tag.
+version = "0.1.0"
```

Comment:
Not critical, but I would probably just start w/ v0.4.0 or something.

Reply:
why? I think its even more confusing lol.

Reply:
Roughly track the other crates in the tokio repo. Not a big deal either way.

Reply:
meh ,lets just keep it `0.1` I think its less confusing.